builtin/logical/pki/crl_test.go

```diff
@@ -1502,3 +1502,119 @@ func TestCRLIssuerRemoval(t *testing.T) {
 	}
 	require.Equal(t, len(afterCRLList), len(crlList))
 }
+
+func TestRevokeExpiredCert(t *testing.T) {
+	t.Parallel()
+
+	coreConfig := &vault.CoreConfig{
```

Comment:
I'd use:

```go
	b, s := CreateBackendWithStorage(t)
```

instead of the cluster setup as it is much faster and nothing we are doing here relies on a full cluster. :-)

Reply:
Alright, I was using another test as a template for mine and picked a test that was setup with a cluster. I will keep this in mind when deciding next time :)

Reply:
Oh, alright, this makes sense, thank you!